sdk/lib/3rdparty/freetype/src/type1/t1gload.c

```diff
@@ -352,7 +397,16 @@
   {
     T1_GlyphSlot            glyph = (T1_GlyphSlot)t1glyph;
     FT_Error                error;
+#ifdef __REACTOS__
+    T1_DecoderRec *decoder = malloc(sizeof(*decoder));
+    if (!decoder)
+      return FT_THROW( Out_Of_Memory );
+/* Ugly but it allows us to reduce the diff */
+#define decoder (*decoder)
```

Comment:
Maybe rename ROS variable to `allocated_decoder` (for example), to better distinguish it from WINE code.
Here and similar cases.

Reply:
Fixed.